.circleci/config.yml

```diff
@@ -39,3 +39,6 @@ workflows:
     jobs:
       - run-tests:
           context: core-team-access
+          filters:
+            tags:
+              only: integration
```

Comment:
I am reading the [circleci tags documentation](https://circleci.com/docs/2.0/configuration-reference/#tags), but I'm not understanding how this relates here. 

We don't have to change anything in the command on line 26 of this file?

Reply:
I think I was conflating the notion of CircleCI testing tags with the golang tag. No tests were run with this setting, so I force pushed an update that tacks the `--tags=integration` onto the end of the run command that starts on line 26.

Reply:
CircleCI is running all the tests again ✔️